enterprise/backend/test/metabase_enterprise/data_editing/api_test.clj

```diff
@@ -23,6 +24,11 @@
 (defn- table-url [table-id]
   (format "ee/data-editing/table/%d" table-id))
 
+(use-fixtures :each
+  (fn [f]
+    (with-redefs [metabase-enterprise.data-editing.api/require-authz? (constantly true)]
```

Comment:
Doing my usual sales pitch for `mt/with-dynamic-fn-redefs`, so this stuff stays parallel friendly.